### PR TITLE
 ArraysUtils to add in our Project

### PR DESCRIPTION
**Pull request by bataborisa**
Should be cool, hope you guys will like it

### CODE REVIEW

**Comment by emiryazicii** [comment]
Thanks for your effort Boris. It looks fine.

**Comment by bataborisa** [comment]
Let's do some magic, my friends, and give big push on branches :)

**Comment by bataborisa** [comment]
Thank you @emiryazicii!

**Review by emiryazicii — APPROVED** [review]

File: src/utils/ArraysUtils.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package utils;
+
+public class ArraysUtils {
+
```
[the remark this answers is not in the document]
Comment (jika17) [reply src/utils/ArraysUtils.java:4]: Why is it still on pending?

[the remark this answers is not in the document]
Comment (emiryazicii) [reply src/utils/ArraysUtils.java:4]: What are you trying to do Jika ? Can you explain so we can understand  the situation and help you .

**Pull request merged**